docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,181 @@
+# Kubernetes Basic Windows User Guide
+This guide gives an example on how to use SMB share(s) in pods running on Windows.
+
+## Prerequisites
+- Setup a Windows cluster following [this guide](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md).
+- A non-cluster Windows 1809 VM with `Full` access to **Storage** under `Cloud API Access Scopes`.
+- An updated personal fork of [kubernetes](https://github.com/kubernetes/kubernetes),[node-driver-registrar](https://github.com/kubernetes-csi/node-driver-registrar) and [gcp-filestore-csi-driver](https://github.com/kubernetes-sigs/gcp-filestore-csi-driver).
+
+## Preparing the forks
+### Kubernetes
+1. Add [Almos98/kubernetes](https://github.com/almos98/kubernetes) as a remote.
+    ```bash
+    git checkout master
+    git remote add demo https://github.com/almos98/kubernetes
+    ```
+2. Apply changes to personal fork.
```

Comment:
You can link to https://github.com/kubernetes/kubernetes/compare/master...almos98:CSI-prototype here to show the changes on the CSI-prototype branch. Same for the other repositories below.

Also, it looks like this branch has two commits on top of master, but these steps are only going to cherry-pick one of those. The easiest thing to do is probably to just squash the commits on the CSI-prototype branch into a single commit. Again, please also check this for the other repositories below.